docs/styles.css

```diff
@@ -701,7 +701,7 @@ footer > div {
 
 /* codelabs */
 .code-lab {
-    margin: 3em 1em 1em;
+    margin: 3em 1em 15px -15px;
```

Comment:
this will definitely improve `go install`

Reply:
Indeed this is the kingpin holding the whole thing together!